subsys/bluetooth/adv_prov/providers/Kconfig.fast_pair

```diff
@@ -21,11 +21,19 @@ if BT_ADV_PROV_FAST_PAIR
 
 config BT_ADV_PROV_FAST_PAIR_ADV_BUF_SIZE
 	int "Size of buffer used for advertising data"
-	range 10 25
-	default 19
+	range 11 26
+	default 20
 	help
-	  Size of the buffer must be big enough to fit Fast Pair advertising
-	  packet.
+	  Size of the buffer must be big enough to fit Fast Pair advertising packet.
+
+	  11 bytes are needed when battery data is not included in Fast Pair advertising
```

Comment:
Maybe we could give exact formula:
`10 + 1.2 * max number of Account Keys encoded in the advertising packets`. If battery notification is enabled we need additional 4 bytes apart from that.